fsspec/implementations/dbfs.py

```diff
@@ -219,6 +226,60 @@ def _open(self, path, mode="rb", block_size="default", **kwargs):
         """
         return DatabricksFile(self, path, mode=mode, block_size=block_size, **kwargs)
 
+    def retry_request(func, retries=5):
+
+        RETRIABLE_EXCEPTIONS = (
+            requests.exceptions.ChunkedEncodingError,
+            requests.exceptions.ConnectionError,
+            requests.exceptions.ReadTimeout,
+            requests.exceptions.Timeout,
+            requests.exceptions.ProxyError,
+            requests.exceptions.SSLError,
+            requests.exceptions.ContentDecodingError,
+
+            aiohttp.client_exceptions.ClientError,
```

Comment:
How does aiohttp enter into this, since all the calls in this module are via requests?

Reply:
I will remove it, copy-pasted and forgot to remove it